### ISSUE TITLE
All Keyboard Shortcuts page (likely) inaccurate

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6117]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.795
    OS Version      : Any
    R Version       : Any

### Steps to reproduce the problem
- Help / Keyboard Shortcuts Help
- In upper left corner of popup, click "See all shortcuts..."

### Describe the problem in detail
- a web page is loaded (`docs/keyboard.htm`)
- this page is static HTML, mostly forgotten or unknown by the RStudio developers, and can (and does) fall out of sync with the currently defined shortcuts

### Describe the behavior you expected
- ideally, this should be autogenerated, or at least have a script to generate it, based on currently defined shortcuts in command.cmd.xml
- minimally, a manual pass through this should be done before shipping 1.3 to ensure it is up-to-date with changes to shortcuts (additions, removals, edits)

**Comment by jmcphers** [comment]
This page has become somewhat redundant in-product since we have a keyboard shortcuts overlay that *is* generated from the command system at runtime doesn't need manually updating. But even if we excise the link from the product we still need an up to date shortcut reference on the web, so no getting around an external artifact we have to keep up to date.

**Comment by gtritchie** [comment]
True, but our overlay is not accessible to screen readers, and doesn't include all the shortcuts, so I was thinking when screen-reader is enabled I'll have the menu command bypass the overlay and just launch the page.  #6118

**Comment by gtritchie** [comment]
Also consider (eventually) getting these verified and updated, too:

https://support.rstudio.com/hc/en-us/articles/200711853-Keyboard-Shortcuts
https://rstudio.com/wp-content/uploads/2016/01/rstudio-IDE-cheatsheet.pdf

**Comment by gtritchie** [comment]
Some shortcuts clash, see https://github.com/rstudio/rstudio/issues/4897

**Comment by ronblum** [comment]
Marking as verified, treating PDF cheat sheets separately.
